fastestimator/util/util.py

```diff
@@ -135,3 +135,19 @@ def strip_suffix(target: Optional[str], suffix: Optional[str]) -> Optional[str]:
     if target[-s_len:] == suffix:
         return target[:-s_len]
     return target
+
+def is_number(s: str):  # pylint: disable=invalid-name
```

Comment:
def is_number(s: str) -> bool:

Reply:
also, maybe we should rename 's' instead of just suppressing the invalid name lint

Reply:
Changing this